theme/recipes/link-recipe.ts

```diff
@@ -0,0 +1,142 @@
+import { defineRecipe } from '@pandacss/dev';
+
+// ts-unused-exports:disable-next-line
+export const linkRecipe = defineRecipe({
+  description: 'The styles for the Link component',
+  className: 'link',
+  jsx: ['Button'],
```

Comment:
?
```suggestion
  jsx: ['Link'],
```

Reply:
For some reason this didn't work for me 🤔 , any ideas? From the docs it says this array is what jsx components the recipe can be applied to, and I'm using a styled jsx button for the `Link` component ...maybe that is why? There is no jsx `Link` component. 🤔

Reply:
Changed the exported component to use a `styled('a')` so works now.

Reply:
[Reading docs](https://panda-css.com/docs/concepts/recipes#advanced-jsx-tracking) I'm less sure what it does than before. Do we need this prop at all? 🤔